uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/directives/leave-request-popup.js

```diff
@@ -36,7 +36,8 @@ define([
           contactId: '<',
           leaveRequest: '<',
           leaveType: '@',
-          userRole: '@'
+          userRole: '@',
+          selectedContactId: '<'
```

Comment:
@igorpavlov Sorry I myself got confused while discussion. I went through your solution and it looks fine to me. No need to change.